transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringSocketChannelConfig.java

```diff
@@ -0,0 +1,242 @@
+package io.netty.channel.uring;
+
+import io.netty.buffer.ByteBufAllocator;
+import io.netty.channel.ChannelException;
+import io.netty.channel.MessageSizeEstimator;
+import io.netty.channel.RecvByteBufAllocator;
+import io.netty.channel.WriteBufferWaterMark;
+import io.netty.channel.socket.SocketChannelConfig;
+
+import java.io.IOException;
+
+
+public class IOUringSocketChannelConfig extends IOUringChannelConfig implements SocketChannelConfig {
```

Comment:
@normanmaurer yeah I thought it would be useful later,  `javaSocket` is required for DefaultSocketChannelConfig, but I dont know how to convert an `linuxSocket` into `javaSocket`